packages/mui-material/src/Select/SelectInput.js

```diff
@@ -487,12 +487,15 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
     ...MenuProps.slotProps?.paper,
   };
 
+  const listboxId = React.useId();
```

Comment:
Let's use `mui-utils` `useId` which is a wrapper on react's `useId` with a fallback

It can be imported as

```js
import { unstable_useId as useId } from '@mui/utils';
```